pkg/schema/examples/hacbs2_valid.json

```diff
@@ -0,0 +1,771 @@
+{
+  "_type": "https://in-toto.io/Statement/v0.1",
+  "predicateType": "https://slsa.dev/provenance/v0.2",
+  "subject": [
+    {
+      "name": "quay.io/hacbs-contract-demo/single-nodejs-app",
+      "digest": {
+        "sha256": "abcbdfd92a75f7bba3ab97538b1324bf4677c1fb3eb82ca59cbd8970b3759b7e"
+      }
+    }
+  ],
+  "predicate": {
+    "builder": {
+      "id": "https://tekton.dev/chains/v2"
+    },
+    "buildType": "https://tekton.dev/attestations/chains/pipelinerun@v2",
```

Comment:
This value is deprecated.

Reply:
Kept to keep a valid version of the JSON